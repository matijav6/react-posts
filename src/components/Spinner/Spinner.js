import React from 'react';
import './Spinner.scss'

export const CustomSpinner = ({isLoading}) => {
    return (
        <>
            {
                isLoading &&
                <div className={'custom-spinner'}>
                    <div className={"progress large"}/>
                </div>
            }
        </>
    );
};
