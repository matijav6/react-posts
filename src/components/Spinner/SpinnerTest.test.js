import renderer from 'react-test-renderer';
import {CustomSpinner} from './Spinner'

test('CustomSpinner renders correctly', () => {
    const tree = renderer
        .create(<CustomSpinner/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
