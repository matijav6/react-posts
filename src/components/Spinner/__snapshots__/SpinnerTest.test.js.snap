// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomSpinner renders correctly 1`] = `null`;
